staging/src/k8s.io/apiserver/pkg/storage/cacher/generic.txt

```diff
@@ -0,0 +1,176 @@
+goos: darwin
```

Comment:
I'm not familiar with this pattern. What serves leaving benchmark results in files? There is no context how those files were created or what they mean. Without analysing them with [benchstat](https://pkg.go.dev/golang.org/x/perf/cmd/benchstat) the reader cannot make conclusions from them.

I think it should be enough to provide the benchstat results in the PR comment or maybe a commit, but please let me know if this is some established pattern I'm not aware.

Reply:
Sorry, this was a mistake, I didn't mean to include this into the PR - Thanks for noticing, I've removed it